src/core/openthread.cpp

```diff
@@ -96,6 +96,19 @@ const uint8_t *otGetExtendedAddress(void)
     return reinterpret_cast<const uint8_t *>(sThreadNetif->GetMac().GetExtAddress());
 }
 
+ThreadError otSetExtendedAddress(const otExtAddress *aExtAddress)
+{
+    ThreadError error = kThreadError_None;
```

Comment:
Would we consider to verify aExtAddress to ensure u/l is 1 and i/g is 0 before making it effect?